### PR TITLE
Added few channels.

### PR DESCRIPTION
**Pull request by Niyasmkkv**

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="AlJazeeraEnglishHD.in" tvg-country="IN" tvg-language="English" tvg-logo="https://cdn.sat.tv/wp-content/uploads/2016/05/AlJazeeraEng.png" group-title="News",AlJazeera English HD
+http://14.199.164.20:4001/play/a0wa/index.m3u8
```
Comment (Dum4G) [inline channels/in.m3u:3]: Quatar channel, no need to stress HD in name

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/in.m3u [hunk 2]
```diff
@@ -321,6 +331,8 @@ http://59c3ec70cfde0.streamlock.net/channel_6/channel6/playlist.m3u8
 https://nicls1-lh.akamaihd.net/i/lst_1@26969/master.m3u8
 #EXTINF:-1 tvg-id="MahaMovie.in" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/hpShFDL.png" group-title="Movies",Maha Movie (576p)
 http://14.199.164.20:4001/play/a0q7/index.m3u8
+#EXTINF:-1 tvg-id="MakkalTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/Sv4bLGX.png" group-title="News",Makkal TV
```
Comment (LaneSh4d0w) [inline channels/in.m3u:334]:
```suggestion
#EXTINF:-1 tvg-id="MakkalTV.in" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/Sv4bLGX.png" group-title="General",Makkal TV
```
Can't really define it as "News" personally. Right now it *seems* it's streaming educational content.

File: channels/in.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/in.m3u:3]: Yup, this channel originates from Qatar (qa), we already have that one by the way.

Comment (Niyasmkkv) [reply channels/in.m3u:3]: can you guys remove this channel add the rest?

**Review by BellezaEmporium — APPROVED** [review]

File: channels/in.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/in.m3u]: Deleted it.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**